### ISSUE TITLE
Keep Research Collection and shipping locations selected

### ISSUE THREAD

**Issue by wyattks** [episphere/connect#560]
On the Specimen link and shipping packaging screens, when a user selects a location, keep that location selected steady (don't have the system revert back to the select default) even if they move to another screen and return.  Only reset the selection to the Select default when the user has logged out of the dashboard and logs back in.

**Comment by wyattks** [comment]
Confirmed - ready for Stage

**Comment by StephanieWeinstein** [comment]
This worked for the collection location but not the shipping location.  I selected "Frederick" from the dropdown shipping location and I put a bag into Box 33.  I then clicked either "home" or "dashboard" and then went back to shipping, and the drop down defaulted back to "select shipping location".  but since I had not logged out, it should have stayed on "Frederick".

I also tested it by moving forward in the shipping dashboard to the second screen (Review Shipment Contents) and then back to the first screen "Packaging", and the shipping location was back to "select shipping location."
